src/qt/guiconstants.h

```diff
@@ -52,4 +52,7 @@ static const int MAX_URI_LENGTH = 255;
 #define QAPP_APP_NAME_TESTNET "Bitcoin-Qt-testnet"
 #define QAPP_APP_NAME_REGTEST "Bitcoin-Qt-regtest"
 
+/* One gigabyte (GB) in bytes */
+static constexpr uint64_t GB_BYTES{1000000000};
```

Comment:
Why is the `LL` suffix no longer needed here? Due to the new initialization style?

Reply:
@laanwj I was wondering about the same thing, and this is what I conclude.

Since C++11, L, LL, UL, ULL should never be needed anymore, as the type of the constant is automatically chosen to be sufficiently large for the provided value, but signed.

It may be necessary to use a U suffix to indicate an unsigned constant, as a 64-bit value (which is larger or equal to 2^63) will by default be of type signed long (long), which would overflow.

Reply:
> Due to the new initialization style?

Yes. [list initialization (since C++11)](https://en.cppreference.com/w/cpp/language/list_initialization):
> Narrowing conversions
> 
> **list-initialization** limits the allowed implicit conversions by prohibiting ... conversion from integer ... to integer type that cannot represent all values of the original, except where source is a constant expression whose value can be stored exactly in the target type

Reply:
Thanks @sipa and @hebasto